docs/run_test/platform.md

```diff
@@ -0,0 +1,44 @@
+# Platform
+
+- [Run on Azure](#run-on-azure)
```

Comment:
Because it's hard to compare this change, so make sure this one is move only, and update content in another commit.